src/pykeen/hpo/hpo.py

```diff
@@ -109,7 +109,7 @@ def _continue_callback(early_stopper: EarlyStopper, result: Union[float, int]) -
         def _stopped_callback(early_stopper: EarlyStopper, result: Union[float, int]) -> None:
             current_epoch = (1 + early_stopper.number_results) * early_stopper.frequency
             trial.set_user_attr(STOPPED_EPOCH_KEY, int(current_epoch))
-            trial.report(result)  # don't include a step because it's over
+            trial.report(result, current_epoch)  # don't include a step because it's over
```

Comment:
now that we're always reporting the epoch, this whole callback system might need a bit of rethinking, but I think its okay for this PR. @mberr @mali-git are we confident in our HPO tests, or could one of you do a local test as well?

Reply:
It would be good to test it locally, too. I will take care of it.